docs/docs/tutorials/get-started/README.mdx

```diff
@@ -0,0 +1,101 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# ⚡ Get started
+
+## Installation
+
+<Tabs>
+
+<TabItem value="docker-compose" label="Docker Compose">
+
+Docker Compose CLI usually comes with [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+```bash
+curl -fsSL https://raw.githubusercontent.com/logto-io/logto/HEAD/docker-compose.yml | docker compose -f - up
+```
+
+After a successful composition, you will see the message like:
+
+</TabItem>
+
+<TabItem value="docker" label="Docker">
+
+1. Prepare a [PostgreSQL](https://www.postgresql.org/)@^14.0 instance。
+2. Pull the image from ghcr.io:
+
+```bash
+docker pull ghcr.io/logto-io/logto:latest
+```
+
+3. Map the container port, e.g., `3001:3001`; and set the following environment variables to the container:
+
+```yml
+ALL_YES: 1
+NO_INQUIRY: 0
+DB_URL_DEFAULT: postgres://username:password@your_postgres_url:5432 # Replace with your Postgres DSN
+```
+
+Finally run the container, you will see the message like:
+
+</TabItem>
+
+<TabItem value="script" label="Download script">
+
+**Prerequisites**
+
+- [Node.js](https://nodejs.org/) `^16.13.0`
+- [PostgreSQL](https://postgresql.org/) `^14.0`
+
+Higher versions usually work but are not guaranteed.
+
+We recommend using a new empty database which is dedicated for Logto, while it's not a hard requirement.
+
+**Download and start**
+
+In your terminal:
+
+```bash
+node -e "$(printf "%s" "$(curl -fsSL https://raw.githubusercontent.com/logto-io/logto/HEAD/install.js)")"
+```
+
+The script will download Logto and create a directory `logto` in the location you ran it. After answering [several simple questions](../../references/core/configuration.md#questions), you will see the message like:
+
+</TabItem>
+
+</Tabs>
+
+```text
+App is running at http://localhost:3001
+```
+
+Heading to <a target="_blank" href="http://localhost:3001/welcome">http://localhost:3001/welcome</a> to continue your Logto journey. Enjoy!
+
+<details>
+  <summary>Configuration (optional)</summary>
+
+Logto uses environment variables for configuration, along with `.env` file support. See [Configuration](../../references/core/configuration.md) for detailed usage and full variable list.
+
+</details>
+
+_Check out [Core Service](../../references/core/README.mdx) if you want more advanced controls or programmatic access to Logto._
+
+## Create an admin account
+
+Open the link and you will see a welcome page that briefly introduces Admin Console. Click "Create Account" to create an admin account.
+
+:::note
+For now, you cannot create multiple admin accounts via Admin Console, and only the first account will have admin access. If you want to create multiple admin accounts, use the Management API instead.
+:::
+
+:::caution
+Once you create the first account, the welcome page will be unavailable.
+:::
```

Comment:
nit: maybe unnecessary? Seems that the welcome page should usually show up only once.

Reply:
there's a confusion that another admin can also use this page to create account

Reply:
unavailable is good to me